templates/workspace_services/health-services/terraform/firewall.tf

```diff
@@ -0,0 +1,35 @@
+resource "azurerm_firewall_application_rule_collection" "health_services_authentication" {
```

Comment:
Is there a reason that you are you not using the pipeline functionality in the the template schema to add the rules? This avoids clashing operations.